kie-server-parent/kie-server-api/src/main/java/org/kie/server/api/commands/CommandScript.java

```diff
@@ -72,7 +75,10 @@
             @XmlElement(name = "get-solvers", type = GetSolversCommand.class),
             @XmlElement(name = "get-solver", type = GetSolverCommand.class),
             @XmlElement(name = "start-solver", type = SolvePlanningProblemCommand.class),
-            @XmlElement(name = "terminate-solver", type = TerminateSolverEarlyCommand.class)
+            @XmlElement(name = "terminate-solver", type = TerminateSolverEarlyCommand.class),
+            @XmlElement(name = "add-problem-fact-change", type = AddProblemFactChangeCommand.class),
+            @XmlElement(name = "add-problem-fact-changes", type = AddProblemFactChangesCommand.class),
+            @XmlElement(name = "get-problem-fact-changes-processed", type = GetProblemFactChangesProcessedCommand.class)
```

Comment:
I'd like the REST api to use the same terminology and even method names as the Solver API as much as reasonable.

The API method is Solver.isEveryProblemFactChangeProcessed(). Instead of

     @XmlElement(name = "get-problem-fact-changes-processed", type = GetProblemFactChangesProcessedCommand.class)

can it be


     @XmlElement(name = "is-every-problem-fact-change-processed", type = IsEveryProblemFactChangeProcessedCommand.class)

?